qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -93,8 +93,25 @@ def decompose_two_qubit_product_gate(special_unitary_matrix):
 
 
 class TwoQubitWeylDecomposition:
-    """Decompose two-qubit unitary U = (K1l⊗K1r).Exp(i a xx + i b yy + i c zz).(K2l⊗K2r) , where U ∈
-    U(4), (K1l|K1r|K2l|K2r) ∈ SU(2), and we stay in the "Weyl Chamber" 𝜋/4 ≥ a ≥ b ≥ |c|
+    r"""
+    Decompose two-qubit unitary
+
+    .. math::
+
+        U = ({K_1}^l \otimes {K_1}^r) \dot e^{(i a xx + i b yy + i c zz)} \dot ({K_2}^l \otimes {K_2}^r)
+
+    where
+
+    .. math::
+
+        U \in U(4)
```

Comment:
I would expect this does not respect linebreaks
```suggestion
        U \in U(4),~
```